addons/account_audit_trail/models/mail_message.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class Message(models.Model):
+    _inherit = 'mail.message'
+
+    account_audit_log_preview = fields.Html(string="Description", compute="_compute_account_audit_log_preview")
+    account_audit_log_move_id = fields.Many2one(
+        comodel_name='account.move',
+        string="Journal Entry",
+        compute="_compute_account_audit_log_move_id",
+        search="_search_account_audit_log_move_id",
+    )
+    show_audit_log = fields.Boolean(compute="_compute_account_audit_log_move_id", search="_search_show_audit_log")
+
+    @api.depends('body', 'subject', 'tracking_value_ids', 'subtype_id')
+    def _compute_account_audit_log_preview(self):
+        for message in self:
+            message_sudo = self.browse(message['id']).sudo().with_prefetch(self.ids)
+            title = message.subject or message.preview
```

Comment:
@william-andre can't remove this line from here, because it's refrenced to if/elif statement.